packages/fxa-react/storybooks.js

```diff
@@ -0,0 +1,42 @@
+const { resolve } = require('path');
+
+const additionalJSImports = {
```

Comment:
Might be nice at some point to extract this from rescripts.js & storybooks.js into a shared config, but didn't want to get into that right now. Maybe if we add a third path here...